website/docs/components/modal/partials/accessibility/accessibility.md

```diff
@@ -17,7 +19,7 @@ Given the Modal is triggered via a keyboard, the dismiss button is first in the
 
 ### Focus order within a complex Modal
 
-If the Modal body contains interactive content, the first element should receive focus first. This is true regardless of how the Modal is triggered; either via a mouse click or via the keyboard.
+If the Modal body contains interactive content, such as input fields, the first element should receive focus first regardless of how the Modal is triggered; either via a mouse click or via the keyboard. This behaviour should be implemented by [automatically setting the focus on a the first element](/components/modal?tab=code#form-within-a-modal-dialog).
```

Comment:
```suggestion
If the Modal body contains interactive content, such as input fields, the first element should receive focus first regardless of how the Modal is triggered; either via a mouse click or via the keyboard. This behaviour should be implemented by [setting the focus on the first element](/components/modal?tab=code#form-within-a-modal-dialog).
```

Reply:
in case, replace `automatically` with `manually` (as a consumer, I have to "manually" do something)

Reply:
yeah, I can see how 'automatically' may be misinterpreted there. I was referring to the fact that the focus itself is automatically set by the browser (and not by a script) based on the attribute, but even if accurate it's not relevant.